preprocessing/generate_wit_inference_data.py

```diff
@@ -0,0 +1,155 @@
+# Copyright 2021 Google LLC
+# 
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# 
+#     https://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import collections
+import json
+import os
+
+import tensorflow as tf
+
+import utils
+
+# `all` configurations.
+INPUT_FILES = 'gs://tabletalk-wit/wit/split/wit_v1.ai.{}.en.recordio-*'
```

Comment:
Add a comment to replace with the actual data folders where someone that checks this repo out will have their data.

Reply:
+1, or make these flags.